Sources/Hummingbird/Middleware/TracingMiddleware.swift

```diff
@@ -25,18 +25,17 @@ import Tracing
 @available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
 public struct HBTracingMiddleware<Context: HBBaseRequestContext>: HBMiddleware {
     private let headerNamesToRecord: Set<RecordingHeader>
+    private let attributes: SpanAttributes?
 
     /// Intialize a new HBTracingMiddleware.
     ///
-    /// - Parameter recordingHeaders: A list of HTTP header names to be recorded as span attributes. By default, no headers
+    /// - Parameters
+    ///     - recordingHeaders: A list of HTTP header names to be recorded as span attributes. By default, no headers
+    ///     - parameters: A list of static parameters added to every span. These could be
     /// are being recorded.
-    public init(recordingHeaders headerNamesToRecord: some Collection<HTTPField.Name>) {
+    public init(recordingHeaders headerNamesToRecord: some Collection<HTTPField.Name> = [], attributes: SpanAttributes? = nil) {
```

Comment:
I assume this is not related to the application context change

Reply:
This is related, in that when we remove the application context we don't have access to some of the application configuration, so cannot update some attributes eg "net.host.name" and "net.host.port". My solution was to allow the user to add some static attributes to the middleware itself.